starter_code/script.js

```diff
@@ -1,9 +1,132 @@
-window.onload = function() {
-  document.getElementById("start-button").onclick = function() {
-    startGame();
+// dessiner la route
+function draw() {
+  var canvas = document.getElementById("road");
+  var ctx = canvas.getContext("2d");
+  // green bg
+  ctx.fillStyle = "#008900";
+  ctx.fillRect(0, 0, 400, 800);
+  // grey road
+  ctx.fillStyle = "#808080";
+  ctx.fillRect(41, 0, 320, 800);
+  // white lines
+  ctx.fillStyle = "#FFFFFF";
+  ctx.fillRect(51, 0, 15, 800);
+  ctx.fillRect(336, 0, 15, 800);
+  // // centered white dashes
+  for (var i = 0; i < 800; i += 40) {
+    ctx.fillStyle = "#FFFFFF";
+    ctx.fillRect(198, i, 4, 30);
+  }
+}
+
+// déclarer la voiture
+var car = {
+  x: 200 - 158 / 8,
+  y: 800 - 500 / 4,
+  moveLeft: function() {
+    this.x -= 25;
+  },
+  moveRight: function() {
+    this.x += 25;
+  }
+};
+
+// dessiner la voiture
+function loadcar(car) {
+  var canvas = document.getElementById("road");
+  var ctx = canvas.getContext("2d");
+  var img = new Image();
+  img.onload = function() {
+    ctx.drawImage(img, car.x, car.y, 158 / 4, 319 / 4);
   };
+  img.src = "images/car.png";
+}
+
+// faire bouger la voiture avec les touches
+document.onkeydown = function(e) {
+  switch (e.keyCode) {
+    case 37:
+      car.moveLeft();
+      console.log("left", car);
+      break;
+    case 39:
+      car.moveRight();
+      console.log("right", car);
+      break;
+  }
+  updateCanvas();
```

Comment:
You should not update the canvas every time a key is pressed, but only when this key is the left or right arrow